modules/runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ConfigurationGenerator.java

```diff
@@ -112,7 +113,13 @@
     /** @since 11.1 */
     public static final String NUXEO_ENVIRONMENT = System.getenv("NUXEO_ENVIRONMENT");
 
-    /** @since 6.0 */
+    /** @since 11.1 */
+    public static final String NUXEO_PROFILES = System.getenv().getOrDefault("NUXEO_PROFILES", EMPTY);
```

Comment:
I'd use `""` instead of `EMPTY`, it's shorter and doesn't require a dependency 😄